hub/apps/toc.yml

```diff
@@ -38,6 +38,8 @@ items:
           href: winui/winui3/create-your-first-winui3-app.md
         - name: "Tutorial: Create a simple photo viewer with WinUI 3"
           href: get-started/simple-photo-viewer-winui3.md
+        - name: "Tutorial: Build a simple photo viewer that targets multiple platforms"
```

Comment:
I would recommend changing the TOC name to either:
Tutorial: Create a simple cross-platform photo viewer
or:
Tutorial: Create a simple photo viewer targeting multiple platforms
"Build" or "Create" is fine. The name here doesn't have to match the article title exactly. We try to make the names shorter to fit in the left column.